pyanaconda/ui/tui/spokes/software_selection.py

```diff
@@ -198,11 +198,6 @@ def refresh(self, args=None):
         threadMgr.wait(THREAD_PAYLOAD)
         self._container = None
 
-        if not self.payload.base_repo:
-            message = TextWidget(_("Installation source needs to be set up first."))
```

Comment:
Wouldn't users miss this message?

Reply:
Good point. I didn't realize that, in TUI, you can enter spokes that are not ready. Fixed.